docs/application/dotnet/get-started/iot/first-app.md

```diff
@@ -0,0 +1,219 @@
+    ```
+
+## Understand Your Application
+
+The C\# code from your first application displays a label centered on the screen, containing the **Welcome to Xamarin Forms!** text. This CrossTemplate1 application created from the template is set up and ready to be built and run by Visual Studio right after you create it, as described above.
+
+The Xamarin.Forms controls used to create the user interface of a Tizen .NET application can be broadly categorized into four groups:
+
+-   `Pages` represent screens within an application. The UI of an application is built from one or more pages and with a navigation mechanism, if needed. The navigation scheme is specified by the `INavigation` interface. Many pages are of the `ContentPage` type, which describes the view of a single screen.
+-   `Layouts` are containers used to compose views into logical structures. Some available types are absolute, grid, relative, and stack layouts; each provides mechanisms, such as orientation, spacing, and padding, to control the layout. The `StackLayout` class is a basic layout where you can simply stack controls on top of, or side-by-side, one another. Layouts can also be bundled with and nested into each other.
+-   `Views` are the controls displayed on the user interface, such as labels, buttons, and text entry boxes.
+-   `Cells` are specialized elements used for items in tables and lists, which help describe how the items must be rendered.
+
+The following shows the portable code portion of the Tizen Xamarin.Forms project **MainPage.xaml** file, generated by the template:
+
+```xml
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="CrossTemplate1.MainPage">
+    <ContentPage.Content>
+        <StackLayout>
+            <Label Text="Welcome to Xamarin.Forms!"
+                VerticalOptions="CenterAndExpand" 
+                HorizontalOptions="CenterAndExpand" />
+        </StackLayout>
+    </ContentPage.Content>
+</ContentPage>
+```
+
+This application is constructed with the following Xamarin.Forms controls:
+
+- There is the `ContentPage` where you set up the view to display. You assign what you want to display to the `Content` property of the ContentPage.
+- The `StackLayout` is added. This layout positions ("stacks") its child elements in a single vertical (default) or horizontal line.
+- The `Label` is added, whose `Text` property is the message you want to display.
+
+The code does not show the instantiation of the `App` class. Since application launching is platform-specific, the launching part, including the instantiation of the `App` class and the declaration of the `Main()` function (required as the entry point of every C\# program), happens in the \<projectname\>.Tizen project, in the matching file there. The following example shows the content of the `<projectname>.Tizen.cs` file:
```

Comment:
Shouldn't this be in bold or code tag or intentionally bold or code tag styling is not applied?

\<projectname\>

```suggestion
The code does not show the instantiation of the `App` class. Since application launching is platform-specific, the launching part, including the instantiation of the `App` class and the declaration of the `Main()` function (required as the entry point of every C\# program), happens in the `\<projectname\>.Tizen` project, in the matching file there. The following example shows the content of the `<projectname>.Tizen.cs` file:
```

Reply:
It's not important information in document.